### PR TITLE
move default python version to 3.10

### PR DESCRIPTION
**Pull request by CaselIT**

### CODE REVIEW

**Comment by kgriffs** [comment]
Added some cython jobs; I know this increases the CI duration but I'd rather be safe than sorry.

**Review by kgriffs — APPROVED** [review]

**Comment by CaselIT** [comment]
thanks!

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -43,27 +41,31 @@ jobs:
...
-          - python-version: 3.9
+          - python-version: "3.9"
             os: ubuntu-20.04
-            toxenv: py39_cython
-          # NOTE(vytas): Quote "3.10" or else it is parsed as a YAML float 3.1.
+            toxenv: py39_cython            
           - python-version: "3.10"
             os: ubuntu-20.04
-            toxenv: py310
+            toxenv: py310
```
Comment (vytas7) [inline .github/workflows/tests.yaml:68]: trailing whitespace

Comment (kgriffs) [reply .github/workflows/tests.yaml]: fixed

**Review by kgriffs — APPROVED** [review]

**Review by vytas7 — APPROVED** [review]
:100:

**Pull request merged**